scene/resources/dynamic_font.cpp

```diff
@@ -118,8 +118,7 @@ Error DynamicFontAtSize::_load() {
 
 	int error = FT_Init_FreeType(&library);
 
-	ERR_EXPLAIN(TTR("Error initializing FreeType."));
-	ERR_FAIL_COND_V(error != 0, ERR_CANT_CREATE);
+	ERR_FAIL_COND_V_MSG(error != 0, ERR_CANT_CREATE, TTR("Error initializing FreeType."));
```

Comment:
The `TTR()` calls are bogus in this file, you can remove them.
`TTR` should never be used outside editor code, it gets replaced by `String()` on non-tools builds.